storage/internal/benchmarks/tmp

```diff
@@ -0,0 +1,13 @@
+// throughput{library="go",api="JSON",op="WRITE",object_size=1024,app_buffer_size=4000,chunksize=16777216,workers=1,crc32c_enabled=true,md5_enabled=false,elapsed_time_us=1602856,bucket_name="brenna-test-bucket",status="OK",timestamp=1677659790,code_version="0.6.1",go_version="go1.19",googlegolangorg_api="v0.110.0",cloudgooglecom_go="v0.107.0",googlegolangorg_grpc="v1.53.0",cloudgooglecom_go_storage="v1.27.0"} 638.86
```

Comment:
Should this file be checked in?

Reply:
No, please ignore this for now. I have been using this PR to debug an issue I'm seeing with the cloud metric API, so I have to do a bit of cleanup (this file and the output metrics)

Reply:
Ok, I finished up here - this should be out of draft now.